helpers/compare.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 const factory = () => {
-    return function(lvalue, rvalue) {
-        const options = arguments[arguments.length - 1];
+    return function(lvalue, rvalue, options) {
+
         var operator;
         var operators;
         var result;
 
-        if (arguments.length < 3) {
+        if (typeof options === undefined) {
```

Comment:
This should be `typeof options === 'undefined'`

Reply:
Also, I would change the error message to say "2 parameters and a block"

Reply:
The thing is that the user passes two arguments and the block is provided by Handlebars.

Reply:
Actually, I will change that now.